### ISSUE TITLE
Features Requested

### ISSUE THREAD

**Issue by Genxster1998** [TachibanaGeneralLaboratories/download-navi#13]
Add Media link Grabber ,Integration With Chromium/Kiwi browser or integrated browser with grabber,Batch File  Downloader ,Site Grabber 

_Sent from my Redmi Note 3 using [FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)_

**Comment by FaizKhan5** [comment]
Also torrent support like in IDM+ in playstore.

**Comment by FaizKhan5** [comment]
It would be great if you implement this because people want one downloader for downloading anything on small internal storage devices.

**Comment by proninyaroslav** [comment]
I will not combine two huge projects into one. This is pointless and makes the app vulnerable to errors.

**Comment by Gymcap** [comment]
I agree with @proninyaroslav, the project should focus on being a really good regular downloaded before torrents are considered.

Plus there are already many open source torrenting clients for android.

**Comment by KaKi87** [comment]
I understand both sides : it would be very practical to have DDL and P2P features in the same app (just like FDM on Windows does), while it seems complicated to implement.
I think the issue should stay open though, so that this feature could be planned on long-term period.

**Comment by jerryn70** [comment]
+1 for website grabber feature

**Comment by KrasnayaPloshchad** [comment]
FDM is now available at [Google Play](https://play.google.com/store/apps/details?id=org.freedownloadmanager.fdm).

**Comment by KrasnayaPloshchad** [comment]
[ADM](https://play.google.com/store/apps/details?id=com.dv.adm) is also add support for torrent.

**Comment by KaKi87** [comment]
Please stop asking for torrent, thanks.
